app/test/simulation_test.go

```diff
@@ -0,0 +1,108 @@
+package app_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simulationtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func init() {
+	simapp.GetSimulatorFlags()
+}
+
+type SimApp interface {
+	GetBaseApp() *baseapp.BaseApp
+	AppCodec() codec.Codec
+	SimulationManager() *module.SimulationManager
+	ModuleAccountAddrs() map[string]bool
+	Name() string
+	LegacyAmino() *codec.LegacyAmino
+	BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock
+	EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.ResponseEndBlock
+	InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain
+}
+
+//var defaultConsensusParams = &abci.ConsensusParams{
+//	Block: &abci.BlockParams{
+//		MaxBytes: 200000,
+//		MaxGas:   2000000,
+//	},
+//	Evidence: &tmproto.EvidenceParams{
+//		MaxAgeNumBlocks: 302400,
+//		MaxAgeDuration:  504 * time.Hour, // 3 weeks is the max duration
+//		MaxBytes:        10000,
+//	},
+//	Validator: &tmproto.ValidatorParams{
+//		PubKeyTypes: []string{
+//			tmtypes.ABCIPubKeyTypeEd25519,
+//		},
+//	},
+//}
```

Comment:
[blocking question]
can we delete these? if we need these, perhaps we can use the testnode.DefaultParams func